addons/web_editor/static/lib/odoo-editor/test/spec/list.test.js

```diff
@@ -5484,8 +5469,8 @@ describe('List', () => {
                     stepFunction: deleteBackward,
                     contentAfter: unformat(`
                         <p class="oe_unbreakable">a[]</p>
-                        <p>ef</p>
                         <ol>
+                            <li>ef</li>
```

Comment:
@Zinston I think this is slightly more correct that the previous one ? It's confusing because the first p is unbreakable, otherwise you'd expect "ef" to land in it. But since the system cannot merge the nodes since the p is unbreakable then I think it's better than ef are kept in their own list item. What do you think ?